render/render.c

```diff
@@ -612,19 +612,22 @@ PictOpValid(CARD8 op)
 }
 
 static int
-SingleRenderComposite(ClientPtr client, xRenderCompositeReq *stuff)
+SingleRenderComposite(ClientPtr client, xRenderCompositeReq *stuff,
+                      Picture srcID, Picture maskID, Picture dstID,
+                      INT16 xSrc, INT16 ySrc, INT16 xMask, INT16 yMask,
+                      INT16 xDst, INT16 yDst)
```

Comment:
Shouldn’t we use the newly introduced parameters when we call `CompositePicture` below?